scripts/funcotator/testing/testFuncotator.sh

```diff
@@ -292,7 +292,7 @@ if [[ $r -eq 0 ]] && ${doRunLargeTests} ; then
     #INPUT=/Users/jonn/Development/gatk/src/test/resources/large/funcotator/regressionTestVariantSet2.vcf
     #INPUT=/Users/jonn/Development/gatk/src/test/resources/large/funcotator/regressionTestHg19Large.vcf
     #INPUT=/Users/jonn/Development/gatk/hg38_trio_liftoverb37.vcf
-    #INPUT=/Users/jonn/Development/gatk/tmp.vcf
+    INPUT=/Users/jonn/Development/gatk/tmp.vcf
```

Comment:
Not sure why this is needed.  What is this stuff?

Reply:
Left over from testing.  I'll remove.